### ISSUE TITLE
Allow users to automatically translate the diary posts to their language

### ISSUE THREAD

**Issue by RobJN** [openstreetmap/openstreetmap-website#2979]
Please add an option for readers to translate OSM diary posts similar to the way that some social media provide an inline button/link to allow you to translate, e.g. tweets.

P.S. The OSMF has supported WeeklyOSM with a DeepL account to help them translate the WeeklyOSM content into multiple languages. At a recent Local Communities chat with the board this idea came up and they seemed supportive of the idea. For the avoidance of doubt, this was a chat and not an instruction from the board. Nevertheless I felt it was a good idea so agreed to open this ticket.

**Comment by xuid0** [comment]
I don't believe copy & pasting my whole suggestion would be a good idea but i wrote: [here on GitHub as a issue] (https://github.com/openstreetmap/openstreetmap-website/issues/1441#issuecomment-2578843087)
